### DISCUSSION TITLE
Filter-"Edit" button with pickable value lists?

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#860] [Ideas]
When you are in html-mode for collection/items and use the Filter-"**Edit**" button, it would be very nice to have **pickable value lists**. One suggestion is to have a boolean property for each field which ensures that selectable lists with distinct values are generated . An error could be triggered on ldproxy-restart if the number of distinct values exceeds a given maximum.

**Comment by azahnen** [comment]
I like the idea, but we will probably not go the full way. What we are currently working on is having such pickable value lists for properties with an enum or codelist in the provider schema (#899).

In theory that would enable your use case, you could just generate codelists using the distinct values both as keys and values.
